docs/freqai-configuration.md

```diff
@@ -204,17 +204,44 @@ If this value is set, FreqAI will initially use the predictions from the trainin
 
 ## Using different prediction models
 
-FreqAI has multiple example prediction model libraries that are ready to be used as is via the flag `--freqaimodel`. These libraries include `Catboost`, `LightGBM`, and `XGBoost` regression, classification, and multi-target models, and can be found in `freqai/prediction_models/`. However, it is possible to customize and create your own prediction models using the `IFreqaiModel` class. You are encouraged to inherit `fit()`, `train()`, and `predict()` to let these customize various aspects of the training procedures.
+FreqAI has multiple example prediction model libraries that are ready to be used as is via the flag `--freqaimodel`. These libraries include `CatBoost`, `LightGBM`, and `XGBoost` regression, classification, and multi-target models, and can be found in `freqai/prediction_models/`.
```

Comment:
we should be cautious about pointing people to internal code directories (like `freqtrade/freqai/prediction_models`). 
That's not available easily depending on the install method (basically in any method which doesn't do a checkout first, like docker or `pip install freqtrade`).

The most likely result will be that we'll have to list all builtin models explicitly someplace in the docs.

Reply:
Good point. I do think listing all in the docs is not a bad thing, although it's something that needs to be maintained. But listing and linking to the lib docs gives users more power over how to use the software so it's definitely worth it.